### PR TITLE
more style fixes

### PR DESCRIPTION
**Pull request by boegel**
one step closer to being able to enable automatic style checking...

### CODE REVIEW

**Comment by boegel** [comment]
@migueldiascosta please review?

File: easybuild/easyconfigs/b/BayeScan/BayeScan-2.1-goolf-1.4.10.eb [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 
 name = 'BayeScan'
 version = '2.1'
 
 homepage = 'http://cmpg.unibe.ch/software/BayeScan/'
-description = """BayeScan aims at identifying candidate loci under natural selection from genetic data, 
+description = """BayeScan aims at identifying candidate loci under natural selection from genetic data,
  using differences in allele frequencies between populations."""
 
-# BayeScan makefile is hardcoded to use g++. In case you want to use a non-gcc toolchain you will need to patch the makefile
```
Comment (boegel) [inline easybuild/easyconfigs/b/BayeScan/BayeScan-2.1-goolf-1.4.10.eb]: no, it's not, the hardcoding is circumvented via `buildopts` deeper down

File: easybuild/easyconfigs/i/I-TASSER/I-TASSER-4.0.eb [hunk 2]
```diff
@@ -25,6 +25,6 @@ sanity_check_paths = {
 
 # You may find it desirable to put this variable in the module file.
 # The command to do so has been put here (commented out) as a convenience.
-#modextravars = { 'IMINTASSERDB' : '/path/to/databases/I-TASSER/%(version)s' }
+# modextravars = { 'IMINTASSERDB' : '/path/to/databases/I-TASSER/%(version)s' }
```
Comment (wpoely86) [inline easybuild/easyconfigs/i/I-TASSER/I-TASSER-4.0.eb]: trim down on the spaces?

**Comment by boegel** [comment]
Thanks for the review @migueldiascosta and @wpoely86!

**Pull request merged**